beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -121,6 +122,14 @@ const MAX_GOSSIP_PROPOSER_SLASHING_QUEUE_LEN: usize = 4_096;
 /// before we start dropping them.
 const MAX_GOSSIP_ATTESTER_SLASHING_QUEUE_LEN: usize = 4_096;
 
+/// The maximum number of queued `SyncCommitteeMessage` objects that will be stored before we start dropping
+/// them.
+const MAX_SYNC_MESSAGE_QUEUE_LEN: usize = 2048;
```

Comment:
Kept this value as 4 * 512(count of valid unique sync committee messages in a slot). Since a sync committee message is useless after the duty slot, I thought it doesn't make much sense to have this as high as the attestation queue length.

On `altair-devnet-1`, getting a max value of 200 for the queue size. Not sure if we should increase it.

Reply:
Yeah I think 4 slots worth of messages should be sufficient. It gives us a little bit of room for dealing with spam, without imposing too high a storage requirement